test/messaging.test.js

```diff
@@ -10,7 +10,8 @@ describe('cap/samples - Messaging', ()=>{
     })
 
     it ('should bootstrap sqlite in-memory db', async()=>{
-        const db = await cds.deploy (_model) .to ('sqlite::memory:')
+        // REVISIT: how to better configure new sqlite
+        const db = await cds.deploy (_model) .to ({impl: '@cap-js/sqlite'}) // 'sqlite::memory:'
```

Comment:
Same

Reply:
I think that's different as it uses cds.test on root and we could configure the plugin within the root package.json.